sdk/src/Services/S3/Custom/Transfer/TransferUtilityDownloadDirectoryRequest.cs

```diff
@@ -82,6 +82,8 @@ internal bool IsSetBucketName()
         /// <summary>
         /// 	Gets or sets the local directory where objects from Amazon S3 will be downloaded.  
 		/// 	If the directory doesn't exist, it will be created.
+        /// 	Kindly note that this is case-sensitive for some platforms/scenarios, so this should
```

Comment:
I appreciate your politeness but "Kindly note" doesn't really fit the tone of API docs.

```
For some platforms like Linux the file system is case-sensitive. The correct casing to the actual path must be used.
```

Reply:
Updated